composer.json

```diff
@@ -12,38 +12,40 @@
         {
             "name": "Samuel Georges",
             "email": "daftspunky@gmail.com"
+        },
+        {
+            "name": "Luke Towers",
+            "email": "octobercms@luketowers.ca"
         }
     ],
     "require": {
-        "php": ">=7.0",
-        "doctrine/dbal": "2.5.*",
-        "erusev/parsedown-extra": "~0.7",
-        "kriswallsmith/assetic": "~1.3",
+        "php": "^7.2",
+        "doctrine/dbal": "^2.6",
+        "erusev/parsedown-extra": "~0.8",
+        "assetic/framework": "dev-2.0/dev",
         "linkorb/jsmin-php": "~1.0",
-        "leafo/scssphp": "~0.6",
-        "symfony/yaml": "~3.2",
+        "symfony/yaml": "^3.4",
         "twig/twig": "~2.0",
         "league/csv": "~8.0",
-        "jenssegers/date": "~3.0",
-        "laravel/tinker": "~1.0"
+        "jenssegers/date": "^3.5",
+        "laravel/tinker": "^1.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "~5.7",
-        "illuminate/database": "5.5.*",
-        "illuminate/filesystem": "5.5.*",
-        "illuminate/queue": "5.5.*",
-        "illuminate/support": "5.5.*",
-        "illuminate/log": "5.5.*",
-        "illuminate/mail": "5.5.*",
-        "illuminate/events": "5.5.*",
-        "illuminate/pagination": "5.5.*",
-        "illuminate/routing": "5.5.*",
-        "illuminate/validation": "5.5.*",
-        "illuminate/view": "5.5.*",
-        "illuminate/cache": "5.5.*",
-        "symfony/dom-crawler": "~3.1",
-        "symfony/css-selector": "~3.1",
-        "meyfa/phpunit-assert-gd": "1.1.0",
+        "phpunit/phpunit": "^8.0|^9.0",
+        "dms/phpunit-arraysubset-asserts": "^0.1.0",
+        "illuminate/database": "^6.0",
+        "illuminate/filesystem": "^6.0",
+        "illuminate/queue": "^6.0",
+        "illuminate/support": "^6.0",
+        "illuminate/log": "^6.0",
+        "illuminate/mail": "^6.0",
+        "illuminate/events": "^6.0",
+        "illuminate/pagination": "^6.0",
+        "illuminate/routing": "^6.0",
+        "illuminate/validation": "^6.0",
+        "illuminate/view": "^6.0",
+        "illuminate/cache": "^6.0",
+        "meyfa/phpunit-assert-gd": "^2.0.0",
```

Comment:
Do you mind grouping all the PHPUnit deps next to each other? Additionally, why do we include all these laravel libraries manually if we include laravel/framework in the main project?

Reply:
@LukeTowers my understanding is that the framework libraries are there so the library can be tested in isolation.

Reply:
@bennothommo fair enough, wasn't sure if the main project was being pulled in to run these tests or not

Reply:
@LukeTowers not with the automated tests on this repo, no. I think @daftspunk has mentioned in the past that the library is sometimes used externally to October.